ruby_on_rails/app_initialisation.md

```diff
@@ -1,61 +1,77 @@
 # Initialise the Rails App
 
-* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
-Run `rbenv versions` to check your installed ruby versions.
+## Default Rails setup
+
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v`
+and update it if you are not. Run `rbenv versions` to check your installed ruby versions.
 Follow the instructions on screen to eventually install it.
 
+* Switch the global Ruby version to the newest one with `rbenv global <ruby-version>` so that the latter commands
+run in the correct context.
+
 * Run `gem install bundler` to install Bundler.
 You may have it already installed, but this command will make sure that you have the latest version.
 
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
-You can do this with `gem update rails`.
+You can do this with `gem update rails`. Beware of beta versions.
```

Comment:
Are they installed with `gem update rails`? did it happen to you with rails 6 beta?

Reply:
Yes.